docs/platform-support.md

```diff
@@ -8,7 +8,7 @@ The most recent version of Celerity aims to support the following environments.
 
 ## SYCL Implementations
 
-* hipSYCL ≥ revision [`db000681`](https://github.com/illuhad/hipSYCL/commit/db000681), with
+* AdaptiveCpp ≥ revision [`v24.06.0`](https://github.com/AdaptiveCpp/AdaptiveCpp/tree/v24.06.0), with
```

Comment:
Should we add a note that we currently don't support the single pass compilation flow?

Reply:
Done!